content/docs/command-reference/remote/modify.md

```diff
@@ -154,6 +154,14 @@ these settings, you could use the following options.
   $ dvc remote modify myremote secret_access_key my-secret_access_key
   ```
 
+- `session_token` - AWS MFA Session Token. May be used (along with
+  `access_key_id` and `secret_access_key`) instead of `credentialpath` when MFA
+  is required:
```

Comment:
```suggestion
- `session_token` - AWS
  [MFA](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa.html)
  session token. May be used (along with `access_key_id` and
  `secret_access_key`) instead of `credentialpath` when MFA is required:
```